src/App.jsx

```diff
@@ -3,26 +3,58 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getAllTodos } from './api';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
   };
 
+  componentDidMount() {
+    getAllTodos()
+      .then(todos => (
+        this.setState({ todos: todos.data.filter(task => task.title
+          && task.userId) })));
+  }
+
+  selectUser = (id) => {
+    this.setState({ selectedUserId: id });
+  }
+
+  clearUser = () => {
+    this.setState({ selectedUserId: 0 });
+  }
+
+  changeTodoStatus = (task) => {
+    this.setState((prevState) => {
+      const newTask = task;
+
+      newTask.completed = !newTask.completed;
+
+      return {
+        todos: [...prevState.todos],
+      };
+    });
+  }
```

Comment:
You can use Map method to your prevState.todos and change 'completed' property of the chosen task inside of it.